### ISSUE TITLE
Use GitHub releases for dists

### ISSUE THREAD

**Issue by jtojnar** [AndrewCrewKuznetsov/xneur-devel#28]
I am updating xneur in NixOS package repository and find it little inconvenient to use the tarballs from repo. If we want to make sure the package is always the same, we need to pin the URL to a certain commit: `https://github.com/AndrewCrewKuznetsov/xneur-devel/raw/f66723feb272c68f7c22a8bf0dbcafa5e3a8a5ee/dists/0.20.0/xneur_0.20.0.orig.tar.gz` It would be much easier if you created a git tag and uploaded the dist tarball to the associated GitHub release. It would also ensure the repository does not bloat with each new release.

**Comment by Ashark** [comment]
I agree. And I think that the archives already contained in repository should be force removed. Because they makes the repository very heavy to clone.
